aztk/client.py

```diff
@@ -108,6 +108,9 @@ def __create_pool_and_job(self, cluster_conf: models.ClusterConfiguration, softw
             start_task=start_task,
             enable_inter_node_communication=True if not cluster_conf.subnet_id else False,
             max_tasks_per_node=4,
+            task_scheduling_policy=batch_models.TaskSchedulingPolicy(
+                node_fill_type=batch_models.ComputeNodeFillType.pack,
```

Comment:
Should we postpone until we know the service can handle this correctly?

Reply:
This is not a fix for anything, but it is better than what we have today. In the simple case, the underlying issue will not be present. Currently the second submitted application is not correctly handled. This delays the issue until the 5th concurrent application at least.